install/README.md

```diff
@@ -223,6 +223,12 @@ of these settings or provide values to mandatory fields.
   - **Type:** `integer`
   - **Default:** `900`
 
+- **`SS_SILENCED_SYSTEM_CHECKS`**:
```

Comment:
What do you think of dropping `SS_` prefix? We are not really consistent but most of the Django related variables don't have it. Also could we move this documentation bit after the `SECRET_KEY` variable?